lib/pystray/_win32.py

```diff
@@ -347,8 +351,8 @@ def _assert_icon_handle(self):
                 None,
                 icon_path,
                 win32.IMAGE_ICON,
-                0,
-                0,
+                128 if Icon.FORCE_LARGE_NOTIFICATION_ICON else 0,
+                128 if Icon.FORCE_LARGE_NOTIFICATION_ICON else 0,
```

Comment:
Is there any reason you would not always want to do this? Are there cases where this causes regressions?

Reply:
My screen dpi value is 240 (scaled 250%). I tested it for different dpi. But the result was similar for all. I setted it 128, because my screen resolution 4k and 64x64 is stil a bit blurry. I calculated icon size for the current dpi (96 -> 32x32, 240 -> 80x80) but the blur did not changed. I also tested my code with 1080p resolution (scaled 100%), no difference for 128x128 or default icon size.

Comparison:

![image](https://user-images.githubusercontent.com/11215801/105834363-6f9d7a00-5fdb-11eb-8ef5-ac037e56b23f.png)